tests/functional/object/test_remote_obc_crud.py

```
@@ -516,3 +522,312 @@ def _calculate_sha256(self, file_path):
+                    Key=test_object_key,
+                )
+                logger.info("Object uploaded successfully")
+
+            # Step 7: Verify objects exist in all mirror backing stores on provider
+            logger.info("Step 7: Verifying objects exist in all mirror backing stores")
+
+            # Note: Verification of objects in backing stores requires NooBaa internal APIs
+            # For now, we verify the object is readable via S3
+            with config.RunWithConfigContext(client_index):
+                # Download and verify
+                download_path = "/tmp/mirror_test_download.bin"
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/mirror_test_download.bin"

(S108)